src/main/java/com/volmit/iris/util/decree/DecreeExecutor.java

```diff
@@ -21,14 +21,45 @@
 import com.volmit.iris.core.tools.IrisToolbelt;
 import com.volmit.iris.engine.framework.Engine;
 import com.volmit.iris.engine.platform.PlatformChunkGenerator;
+import com.volmit.iris.util.format.C;
 import com.volmit.iris.util.plugin.VolmitSender;
+import org.bukkit.World;
+import org.bukkit.entity.Player;
 
 public interface DecreeExecutor {
     default VolmitSender sender()
     {
         return DecreeContext.get();
     }
 
+    default Player player()
+    {
+        if (sender().isPlayer()) {
+            return sender().player();
```

Comment:
Dont null check here. A NPE will fire either way if its not a player. Simply return sender.player() it makes the stack trace easier to read

Reply:
Five